src/services/fetcher.ts

```diff
@@ -0,0 +1,61 @@
+import axios from "axios";
+import { ParamsType } from "../types";
+
+export const baseURL = "https://test-front.framework.team";
+const client = axios.create({ baseURL });
+
+export const getAuthors = async () => {
+  const result = await client.get("authors");
+  return result.data;
+};
+
+export const getLocations = async () => {
+  const result = await client.get("locations");
+  return result.data;
+};
+
+export const getAllPaintings = async () => {
```

Comment:
ниже  субъективщина, я пишу так
```suggestion
export function getAllPaintings () {
  return client.get("/paintings").then(r => r.data);
};
```

Reply:
+ я не люблю объявлять лямбды на верхнем уровне(как будто код читаемее становится)

Reply:
и ещё у тебя ни в одном фетчере не стоит ретёрн тип, это точно плохо